packages/strapi/Dockerfile

```diff
@@ -1,9 +1,8 @@
 # Development Dockerfile
 
-FROM node:18-alpine
-# Installing libvips-dev for Sharp compatibility
-RUN apk update && apk add --no-cache build-base gcc autoconf automake zlib-dev libpng-dev nasm bash vips-dev
-ARG NODE_ENV=development
```

Comment:
If the ARG instruction is deleted then probably the following ENV instruction that uses it should be deleted too?

Reply:
Yes, it not used, so I remove it